app/controllers/preprints/provider/preprint-detail.js

```diff
@@ -124,6 +139,9 @@ export default Controller.extend({
 
             this._saveAction(action, filter);
         },
+        closeReviewerFeedback(userActivity) {
```

Comment:
This name seems misleading -- it doesn't close anything.

Reply:
what about 'setUserEnteredReview' ?

Reply:
Sure, your call. As long as the name describes what the thing is or what the thing does.